touchforms/backend/test_xformplayer.py

```diff
@@ -75,7 +75,7 @@ def mock_query(query_function, criteria):
             self.xform,
             session_data=self.session_data,
             form_context={
-                'all_case_ids': ['legolas', 'gimli'],
+                'all_case_ids': ['legolas', 'sauron'],
```

Comment:
yeah, that's exactly right. I think the better fix would be to do the thing I suggest in the comment, and the even better fix would be to figure out why the mobile makes that call, but both of those seemed out of scope